satellite/metainfo/config.go

```diff
@@ -21,6 +21,7 @@ const (
 // RSConfig is a configuration struct that keeps details about default
 // redundancy strategy information
 type RSConfig struct {
+	MaxSegmentSize   memory.Size `help:"maximum segment size (in bytes)" default:"64MiB"`
```

Comment:
nit: "(in bytes)" is somewhat redundant because `memory.Size` handles size conversion automatically